src/redux/wallet/saga.js

```diff
@@ -225,7 +225,9 @@ function* updateTransactionRequest(action) {
    && item.type === transaction.type
    && (item.address === transaction.from || item.address === transaction.to));
   _.each(foundTokens, (token) => {
-    addTokenTransactions(token, [transaction]);
+    const isSender = token.address === transaction.from;
+    const newTransaction = parseDataUtil.processTransaction(transaction, isSender);
```

Comment:
Here as well. I don't see any reason to calculate isSender outside and pass it in.

Reply:
If you calculate inside you wouldn't need to write the same line of code twice(The other is on top of this PR).